x-pack/plugins/security_solution/cypress/e2e/detection_alerts/alert_table_hover_actions.cy.ts

```diff
@@ -0,0 +1,177 @@
+       * in how `_id` is supplied to the security solution by trigger actions table
+       *
+       * */
+      waitForAlerts();
+      cy.get(ID_COLUMN_VALUES).first().trigger('mouseover', { timeout: 5000 });
+    });
+
+    it('Filter For', () => {
+      cy.get(ID_COLUMN_VALUES)
+        .eq(0)
+        .then(($el) => {
+          const idTextValue = $el.text();
+          cy.get(HOVER_ACTIONS.FILTER_FOR).should('exist');
+          cy.get(HOVER_ACTIONS.FILTER_FOR).trigger('click', { force: true });
+          cy.get(GLOBAL_SEARCH_BAR_FILTER_ITEM).should('contain.text', `_id: ${idTextValue}`);
+          cy.get(GLOBAL_SEARCH_BAR_FILTER_ITEM_DELETE).trigger('click');
+          cy.get(GLOBAL_SEARCH_BAR_FILTER_ITEM).should('not.exist');
+        });
+    });
+
+    it('Filter Out', () => {
+      cy.get(ID_COLUMN_VALUES)
+        .eq(0)
+        .then(($el) => {
+          const idTextValue = $el.text();
+          cy.get(HOVER_ACTIONS.FILTER_OUT).should('exist');
+          cy.get(HOVER_ACTIONS.FILTER_OUT).trigger('click', { force: true });
+          cy.get(GLOBAL_SEARCH_BAR_FILTER_ITEM).should('contain.text', `NOT _id: ${idTextValue}`);
+          cy.get(GLOBAL_SEARCH_BAR_FILTER_ITEM_DELETE).trigger('click');
+          cy.get(GLOBAL_SEARCH_BAR_FILTER_ITEM).should('not.exist');
+        });
+    });
+
+    it('Add To Timeline', () => {
+      cy.get(ID_COLUMN_VALUES)
+        .eq(0)
+        .then(($el) => {
+          const idTextValue = $el.text();
+          cy.get(HOVER_ACTIONS.ADD_TO_TIMELINE).should('exist');
+          cy.get(HOVER_ACTIONS.ADD_TO_TIMELINE).trigger('click', { force: true });
+          openActiveTimeline();
+          cy.get(TIMELINE_DATA_PROVIDERS_CONTAINER).should('be.visible');
+          cy.get(TIMELINE_DATA_PROVIDERS_CONTAINER).should('contain.text', idTextValue);
+          closeTimelineUsingCloseButton();
+        });
+    });
+  });
+
+  context('Host Column', { testIsolation: false }, () => {
+    before(() => {
+      visit(ALERTS_URL);
+      waitForAlerts();
+    });
+    beforeEach(() => {
+      cy.get(HOST_NAME).first().trigger('mouseover', { timeout: 5000 });
+      // we want to trigger second set of hover actions on host column
+      // first is on _id column
+      cy.get(CELL_EXPAND_VALUE).first().should('exist');
+      cy.get(CELL_EXPAND_VALUE).first().trigger('click', { force: true });
+      cy.get(CELL_EXPANSION_POPOVER).should('be.visible');
+    });
+
+    it('TopN', () => {
+      cy.get(HOVER_ACTIONS.SHOW_TOP).should('be.visible').trigger('click');
+      waitForTopNHistogramToLoad();
+      cy.get(TOP_N_ALERT_HISTOGRAM).should('be.visible');
+      cy.get(TOP_N_CONTAINER_CLOSE_BTN).trigger('click', { force: true });
+      cy.get(TOP_N_ALERT_HISTOGRAM).should('not.exist');
+      // close the expanded popover
+      cy.get(CELL_EXPAND_VALUE).first().click({ force: true });
+    });
+
+    it('Host Summary', () => {
```

Comment:
if I understand correctly, this test is basically testing that the flyout shows up when clicking on an hover action, then that it disappears when clicking on the close (cross) button?

I feel like testing the closing is unnecessary as it's being handled by the EuiFlyout isn't it?

Also I feel like testing the opening when clicking on the hover action could be bundled with a test that actually tests something within the flyout?